Add admin page for creating new organizations with facilities attached

ClydeDroid opened the pull request:
## Related Issue or Background Info

Resolves #450 

## Changes Proposed

- Changes `createOrganization` mutation to include facility information
- Adds an admin page with a link to a form to create an organization w/ facility

## Additional Information

- How do I limit the `/admin` route to admin users only? I don't see any permissions stuff on the frontend
- Did I miss anything?

## Screenshots / Demos
![image](https://user-images.githubusercontent.com/9121162/104076051-a5fc8c00-51c9-11eb-94c6-341380a078a9.png)
![image](https://user-images.githubusercontent.com/9121162/104076074-b9a7f280-51c9-11eb-8858-0d9f92a073d8.png)

timothybest-usds commented on frontend/src/app/admin/Organization/OrganizationFormContainer.tsx:
```diff
@@ -0,0 +1,265 @@
+import OrganizationForm from "./OrganizationForm";
+import { Redirect } from "react-router-dom";
+
+const GET_FACILITY_QUERY = gql`
```
I this this query can be removed and `getFacilityData` can always return an empty Facility

timothybest-usds reviewed (commented):
> How do I limit the /admin route to admin users only? I don't see any permissions stuff on the frontend
We don't have any permission stuff on the frontend yet. One way we could handle this is to pass an `isAdmin` flag in the response for the query for `whoami`

ClydeDroid commented:
@TimBest I went ahead and added an `isAdmin` field to the `whoami` query, and only include the `/admin` routes for admin users. I also changed the `getFacility` query to just get the device types, which I still need to pass through to the `ManageDevices` component (I think?) Let me know what you think!
